HOW_TO.md

```diff
@@ -0,0 +1,53 @@
+## How to join the network?
+
+RPC address would be provided here shortly.
+
+We recommend you to use [near-cli documentation](https://docs.near.org/tools/near-cli), e.g. how to [create an account](https://docs.near.org/tools/near-cli#near-create-account).
+
+StatelessNet is a sandbox created for load testing purposes.
+The traffic from mainnet may be forwarded to StatelessNet.
+It means that all the existing mainnet accounts would be already occupied.
+It will not affect the activity on mainnet in any case.
+
+If you need some tokens for your activity, fill out the form: [the link to the form will be provided shortly].
+
+## How to become a validator?
```

Comment:
I would subdivide into three sections
* how to build the binary
* how to get the config and genesis
* how to get the backup 

And I would also add a link to the page with instructions for setting up a mainnet validator

Reply:
Also giving a commit seems a bit weird, perhaps we can at least tag it or create a branch for it? WDYT?

Reply:
Agree with everything!
I created the tag, added the tag and the link to the validator docs to the landing.
I like the idea to split the details into 3 sections you suggested, but I need your help because I don't have enough context to fill in each section

Reply:
* how to build the binary - Can you check what is the recommended way in the public documentation and then just add the sv feature to it? I would expect it to like something like this: 
```cargo build -p neard --release --features statelessnet_protocol```
* config and genesis - @andrei-near I have no clue but I hope @andrei-near will put it in a cloud and give us the links
* backup - @andrei-near again :)

Reply:
How would updating to the latest version work with tags?
I'm no git expert, but I read that there can be only one tag with some name. Are there going to be multiple tags for subsequent versions?

I think it would be more user friendly to have a branch. Then the user can just run `git pull origin stake-wars-iv` to get the latest version (Or have a script do it every X hours)

Reply:
@jancionear tag is just a pointer. We can change the tag as well (drop and recreate it to the desired commit)